sdks/python/apache_beam/examples/snippets/snippets_test.py

```diff
@@ -1261,6 +1261,10 @@ def extract_output(self, sum_count):
         (sum, count) = sum_count
         return sum / count if count else float('NaN')
 
+      def combine(self, accumulator):
+        # No-op
```

Comment:
do we want to change the TypeScript snippet?

Reply:
TypeScript doesn't currently support compact, and I did not think it was worth adding a callout that its unsupported since its likely to go out of date

Reply:
oh. I see.